tsweb/debug.go

```diff
@@ -48,16 +48,11 @@ func Debugger(mux *http.ServeMux) *DebugHandler {
 	}
 	mux.Handle("/debug/", ret)
 
-	// Register this one directly on mux, rather than using
-	// ret.URL/etc, as we don't need another line of output on the
-	// index page. The /pprof/ index already covers it.
-	mux.Handle("/debug/pprof/profile", BrowserHeaderHandler(http.HandlerFunc(pprof.Profile)))
```

Comment:
was that BrowserHeaderHandler important?

Reply:
Yes, but the debughandler's ServerHTTP does it inline, so no need for a dedicated extra wrapping.